src/TropicalGeometry/hypersurface.jl

```diff
@@ -210,15 +210,41 @@ end
 @doc raw"""
     dual_subdivision(TropH::TropicalHypersurface)
 
-Return the dual subdivision of `TropH`.  Raises an error, if neither it nor the the internal polymake object is cached.
+Return the dual subdivision of `TropH`.  Raises an error, if neither it nor the
+the internal polymake object is cached.
```

Comment:
Nitpick from an outsider (so this is by no means a blocker, double so since this is just reformatting existing text): I find the second sentence rather technical and I am not sure what it even is telling me. My uninformed guess would be: this function can not actually compute a dual subdivision; it can only return one if it is already set somehow. If that's what it means, then perhaps it could be stated as such? If it means something else, well, perhaps that could be explained better?